docs/getting_started.rst

```diff
@@ -30,7 +30,8 @@ file, place it in the ``cfme_tests`` repository (along ``conftest.py``):
 
   . ./.cfme_tests/bin/activate
   python scripts/disable-bytecode.py
-  PYCURL_SSL_LIBRARY=nss pip install -Ur ./requirements.txt
+  pip install --upgrade pip
+  PYCURL_SSL_LIBRARY=nss pip install -Ur ./requirements.txt --no-cache-dir
```

Comment:
No no, I'm not saying you _should_ put --no-cache-dir, I'm saying that when I ran it, to ensure I wasn't using a cahced version, ie to force a rebuild, that's what I did, the upgrade pip is a good idea, but as this is a getting started guide, wouldn't one expect them to have a fresh pip anyway?